src/content/1.7/basics/installation/_index.md

```diff
@@ -23,16 +24,65 @@ Here is a selection of free AMP installers:
 * [MAMP](https://www.mamp.info/) (Mac OS X)
 * [Laragon](https://laragon.org/) (Windows)
 
+To install LAMP on your computer follow these steps (tested on Debian Buster).
+- Update your system
+  - `apt update`
+- Install MySQL
+  - `apt install default-mysql-server default-mysql-client`
+- Install Apache server
+  - `apt install apache2`
+- Install PHP
+  - `apt install libapache2-mod-php7.3 php7.3 php7.3-common php7.3-curl php7.3-gd php7.3-imagick php7.3-mbstring php7.3-mysql php7.0-json php7.3-xsl php7.3-intl php7.3-zip`
 
+## Creating a database for your shop
 
-## Creating a database for your local shop
+If you are installing PrestaShop on a web server, then you must create the database and give access to a privileged user.
+This user is the one which will be set during the installation process.
 
-Open the phpMyAdmin tool using your browser. Its location depends on the AMP pack you chose:
+### With phpMyAdmin
 
-* http://127.0.0.1/phpmyadmin (XAMPP, WampServer, MAMP),
-* http://127.0.0.1/mysql (EasyPHP)
+We supposed you have a root access to `phpMyAdmin`, and you're using a 4.x.
+
+ * Sign in to `phpMyAdmin` as the root user
+ * Click `User accounts`, and then click on `Add user account`
+ * Fill the `User name` and the `Password`
+ * In the `Database for user account`, select `Create database` and `Grant all privileges`
+ * Create user and database and make sure the COLLATION of your database is `utf8mb4_general_ci`
+ 
+### From the command line
+
+The database must be created with UTF-8 encoding (utf8mb4_general_ci).
+For information on installation and configuring MySQL see:
+[https://dev.mysql.com/doc/refman/5.6/en/](https://dev.mysql.com/doc/refman/5.6/en/)
+
+```bash
+$ mysql -u adminusername -p
+Welcome to the MariaDB monitor.  Commands end with ; or \g.
+Your MariaDB connection id is 12508
+Server version: 10.3.17-MariaDB-0+deb10u1 Debian 10
+
+Copyright (c) 2000, 2018, Oracle, MariaDB Corporation Ab and others.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
+
+MariaDB [(none)]> CREATE DATABASE prestashop COLLATE utf8mb4_general_ci;
+Query OK, 1 row affected (0.000 sec)
+
+MariaDB [(none)]> GRANT ALL PRIVILEGES ON prestashop.* TO "prestashopuser"@"hostname" IDENTIFIED BY "password";
+Query OK, 0 rows affected (0.001 sec)
+
+
+MariaDB [(none)]> FLUSH PRIVILEGES;
+Query OK, 0 rows affected (0.001 sec)
+```
+
+The example above shows:
+ - the `adminusername` is connected with `root` permissions
+ - `prestashop` is the name of the wanted database
+ - `prestashopuser` is the user name of the database, but please don't forget to use something you're the only one to know
+ - `hostname` is usually the localhost (`127.0.0.1` or `localhost`), if you don't know the value, check with a system adminitrator
+ - `password` must be something strong and of course, only you know it
```

Comment:
Suggestion:

    Connect as root to your MySql server. In this example our root user is called `adminusername`:

    ```bash
    $ mysql -u adminusername -p
    ```

    Create the database and give it a name like "prestashop":

    ```bash
    > CREATE DATABASE prestashop COLLATE utf8mb4_general_ci;
    ```

    Grant privileges to that database to a new user (the one that PrestaShop will use to connect to the database). Let's call it "prestashopuser".

    ```bash
    > GRANT ALL PRIVILEGES ON prestashop.* TO "prestashopuser"@"hostname" IDENTIFIED BY "somepassword";
    ```

    In the example above,

    - `prestashop` is the name of the new database
    - `hostname` is usually localhost (`127.0.0.1` or `localhost`), if you don't know the value, check with a system adminitrator
    - `somepassword` must be a strong password and of course, only known by you

    Finally, flush privileges:

    ```bash
    > FLUSH PRIVILEGES;
    ```